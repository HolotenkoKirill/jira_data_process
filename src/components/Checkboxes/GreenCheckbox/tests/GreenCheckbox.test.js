import React from 'react';
import {shallow} from 'enzyme';
import GreenCheckbox from '../GreenCheckbox';
import serializer from 'enzyme-to-json/serializer';

expect.addSnapshotSerializer(serializer);

describe('<GreenCheckbox />', () => {
    it('renders GreenCheckbox component', () => {
        const component = shallow(<GreenCheckbox />).dive();

        expect(component).toMatchSnapshot();
    });
});
