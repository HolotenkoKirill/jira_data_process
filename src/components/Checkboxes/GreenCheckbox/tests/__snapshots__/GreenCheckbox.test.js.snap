// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GreenCheckbox /> renders GreenCheckbox component 1`] = `
<WithStyles(ForwardRef(Checkbox))
  classes={
    Object {
      "checked": "Component-checked-2",
      "root": "Component-root-1",
    }
  }
  color="default"
/>
`;
